### ISSUE TITLE
WorkflowInstance properties

### ISSUE THREAD

**Issue by dimademi** [danielgerlag/workflow-core#716]
Hey! 
I didn't find any documentation for what is the `WorkflowInstance.Reference` property.

What is it used for? How can i utilize it? 
Furthermore, is there any way to set a description with fluent api?

**Comment by xlegalles** [comment]
I agree: I just discover the library, cannot find any documentation about it... and dig into the source code to understand
